config_center/apollo/impl.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 import (
+	gxset "github.com/dubbogo/gost/container/set"
```

Comment:
I used the `set` interface defined in `"github.com/emirpasic/gods/sets"` as func's return parameter. Should we have a talk about the set data structure? I think we can either rewrite our set container structure or use `"github.com/emirpasic/gods/sets"`

Reply:
Using `github.com/emirpasic/gods/sets` and then remove `github.com/dubbogo/gost/container/set` is better.